core/src/main/scala/org/apache/spark/ui/jobs/StagesTab.scala

```diff
@@ -35,7 +35,7 @@ private[ui] class StagesTab(parent: SparkUI) extends SparkUITab(parent, "stages"
   attachPage(new StagePage(this))
   attachPage(new PoolPage(this))
 
-  def isFairScheduler: Boolean = progressListener.schedulingMode == Some(SchedulingMode.FAIR)
+  def isFairScheduler: Boolean = progressListener.schedulingMode.contains(SchedulingMode.FAIR)
```

Comment:
Up to my knowledge, `contains` on `Option` breaks Scala 2.10 build (see https://github.com/apache/spark/pull/15393). I guess we are going to drop the support but not yet.